terranova/commands/binds.py

```diff
@@ -117,7 +130,16 @@ def init(path: str | None, migrate_state: bool, no_backend: bool, reconfigure: b
                 upgrade=upgrade,
             )
         except sh.ErrorReturnCode as err:
-            raise Exit(code=err.exit_code) from err
+            error = Exit(code=err.exit_code)
+            if fail_at_end:
+                errors = True
+                Log.failure(error)
+            else:
+                raise error from err
```

Comment:
```suggestion
            errors = True
            if not fail_at_end:
                break
```

The error itself isn't relevant since we print the terraform stderr.